docs/en/engines/table-engines/integrations/nats.md

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_position: 14
+sidebar_label: NATS
+---
+
+# NATS Engine {#redisstreams-engine}
```

Comment:
Let's also mention setting `stream_like_engine_insert_queue` and why it is needed.